### PR TITLE
RC: Default maintenance windows for Essentials databases

### PR DESCRIPTION
**Pull request by cmilesb**
DOC-3985

### CODE REVIEW

File: content/operate/rc/resilient-apps.md [hunk 1]
```diff
@@ -52,6 +52,8 @@ Redis maintains your Redis Cloud subscriptions and databases as needed to ensure
 
 For stable apps, you may want to control when Redis can perform maintenance on your databases. For Redis Cloud Pro subscriptions, you can [set manual maintenance windows]({{< relref "/operate/rc/subscriptions/maintenance/set-maintenance-windows" >}}) to ensure non-urgent maintenance will occur at set times. Configuring or altering the maintenance window will not have any impact on your subscription or databases.
 
+Redis Cloud Essentials databases have a set maintenance window based on the region where it is located. The maintenance window for Redis Cloud Essentials databases is **daily from 12 AM to 6 AM** in the region's local time zone. See [Essentials maintenance]({{< relref "/operate/rc/subscriptions/maintenance#redis-cloud-essentials" >}}) for more information.
```
Comment (andy-stark-redis) [inline content/operate/rc/resilient-apps.md:55]:
```suggestion
A Redis Cloud Essentials database has a set maintenance window based on the region where it is located. The maintenance window for Redis Cloud Essentials databases is **daily from 12 AM to 6 AM** in the region's local time zone. See [Essentials maintenance]({{< relref "/operate/rc/subscriptions/maintenance#redis-cloud-essentials" >}}) for more information.
```

File: content/operate/rc/subscriptions/maintenance/set-maintenance-windows.md [hunk 2]
```diff
@@ -14,7 +14,9 @@ weight: $weight
...
 
-By default, Redis will perform maintenance automatically while limiting service disruption as much as possible. If you want to control when Redis performs maintenance for a Redis Cloud Pro subscription, you can [set manual maintenance windows](#set-manual-maintenance-windows) to ensure non-urgent maintenance will occur at set times. Configuring or altering the maintenance window will not have any impact on your subscription or databases.
+For Redis Cloud Pro plans, Redis will perform maintenance automatically while limiting service disruption as much as possible. If you want to control when Redis performs maintenance for a Redis Cloud Pro subscription, you can [set manual maintenance windows](#set-manual-maintenance-windows) to ensure non-urgent maintenance will occur at set times. Configuring or altering the maintenance window will not have any impact on your subscription or databases.
+
+Redis Cloud Essentials databases have a set maintenance window based on the region where it is located. See [Essentials maintenance]({{< relref "/operate/rc/subscriptions/maintenance#redis-cloud-essentials" >}}) for more information.
```
Comment (andy-stark-redis) [inline content/operate/rc/subscriptions/maintenance/set-maintenance-windows.md:19]:
```suggestion
A Redis Cloud Essentials database has a set maintenance window based on the region where it is located. See [Essentials maintenance]({{< relref "/operate/rc/subscriptions/maintenance#redis-cloud-essentials" >}}) for more information.
```

**Review by andy-stark-redis — APPROVED** [review]
Couple of minor issues, but language is fine otherwise.

File: content/operate/rc/resilient-apps.md [hunk 3]
```diff
@@ -52,6 +52,8 @@ Redis maintains your Redis Cloud subscriptions and databases as needed to ensure
 
 For stable apps, you may want to control when Redis can perform maintenance on your databases. For Redis Cloud Pro subscriptions, you can [set manual maintenance windows]({{< relref "/operate/rc/subscriptions/maintenance/set-maintenance-windows" >}}) to ensure non-urgent maintenance will occur at set times. Configuring or altering the maintenance window will not have any impact on your subscription or databases.
 
+A Redis Cloud Essentials database has a set maintenance window based on the region where it is located. The maintenance window for Redis Cloud Essentials databases is **daily from 12 AM to 6 AM** in the region's local time zone. See [Essentials maintenance]({{< relref "/operate/rc/subscriptions/maintenance#redis-cloud-essentials" >}}) for more information.
```
Comment (NoamSternRedis) [inline content/operate/rc/resilient-apps.md:55]: just a reference from here is good enough. 
We can remove the sentence: "The maintenance window for Redis Cloud Essentials databases is **daily from 12 AM to 6 AM** in the region's local time zone."

File: content/operate/rc/subscriptions/maintenance/set-maintenance-windows.md [hunk 4]
```diff
@@ -14,7 +14,9 @@ weight: $weight
...
 
-By default, Redis will perform maintenance automatically while limiting service disruption as much as possible. If you want to control when Redis performs maintenance for a Redis Cloud Pro subscription, you can [set manual maintenance windows](#set-manual-maintenance-windows) to ensure non-urgent maintenance will occur at set times. Configuring or altering the maintenance window will not have any impact on your subscription or databases.
+For Redis Cloud Pro plans, Redis will perform maintenance automatically while limiting service disruption as much as possible. If you want to control when Redis performs maintenance for a Redis Cloud Pro subscription, you can [set manual maintenance windows](#set-manual-maintenance-windows) to ensure non-urgent maintenance will occur at set times. Configuring or altering the maintenance window will not have any impact on your subscription or databases.
+
+A Redis Cloud Essentials database has a set maintenance window based on the region where it is located. See [Essentials maintenance]({{< relref "/operate/rc/subscriptions/maintenance#redis-cloud-essentials" >}}) for more information.
```
Comment (NoamSternRedis) [inline content/operate/rc/subscriptions/maintenance/set-maintenance-windows.md:19]: maybe put it in a note instead of a paragraph

**Review by NoamSternRedis — CHANGES_REQUESTED** [review]

File: content/operate/rc/subscriptions/maintenance/_index.md [hunk 5]
```diff
@@ -33,19 +43,21 @@ Redis will notify users by email when maintenance starts and ends. For more deta
...
 
 Urgent maintenance refers to any activity that could affect service and cannot wait for scheduling. This includes applying urgent security patches.
 
 Redis can perform urgent maintenance at any time, even if you have set a manual maintenance window or have temporarily [skipped maintenance]({{< relref "/operate/rc/subscriptions/maintenance/set-maintenance-windows#skip-maintenance-temporarily" >}}). Redis will notify you by email when urgent maintenance starts and ends.
 
-## Notifications
+### Major upgrades
 
-Redis will notify you by email when maintenance starts and ends. If Redis needs an action from you to start maintenance, Redis will notify you with a reasonable amount of time before planned maintenance.
+We won't schedule major upgrades or upgrades that might include breaking changes until you opt-in to those changes. You can opt-in to specific major changes on the [subscription page]({{< relref "/operate/rc/subscriptions/view-pro-subscription" >}}).
```
Comment (NoamSternRedis) [inline content/operate/rc/subscriptions/maintenance/_index.md:56]: Suggestion - replace "You can opt-in to specific major changes on the [subscription page]" to "You can control your Redis major versions on the [subscription page]"

Comment (cmilesb) [reply content/operate/rc/subscriptions/maintenance/_index.md:56]:
```suggestion
We won't schedule major upgrades or upgrades that might include breaking changes until you opt-in to those changes. You can control your Redis major version on the [subscription page]({{< relref "/operate/rc/subscriptions/view-pro-subscription" >}}).
```

**Pull request merged**